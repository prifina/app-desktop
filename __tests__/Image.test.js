import React from "react";
import renderer from "react-test-renderer";
import ScreenImages from "../src/components/Image";
import { ThemeProvider } from "@blend-ui/core";
// import {imageFile} from '../src/assets/maxresdefault1.png';


test("Image renders correctly", () => {
    const tree = renderer.create(
        <ThemeProvider>
            <ScreenImages src={"https://picsum.photos/200/200"} />
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});
