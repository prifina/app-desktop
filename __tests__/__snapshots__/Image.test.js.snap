// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1w52cfi-0 fESRGN"
>
  <img
    alt="image"
    className="sc-bdfBwQ eCgEDi"
    height="auto"
    src="https://picsum.photos/200/200"
  />
</div>
`;